src/main/scala/org/camunda/feel/impl/builtin/NumericBuiltinFunctions.scala

```diff
@@ -164,4 +166,10 @@ object NumericBuiltinFunctions {
       case List(ValNumber(n), ValNumber(scale)) =>
         round(n, scale, RoundingMode.HALF_DOWN)
     })
+
+  private def randomNumberFunction =
+    builtinFunction(params = List(), invoke = {
+      case List() =>
+        ValNumber(Random.nextFloat())
```

Comment:
:wrench: We could use `nextDouble()` instead of `nextFloat()`. Numbers are represented as `BigDecimal`.